src/apps/dictionaries/components/ViewDictionariesPage.tsx

```diff
@@ -1,20 +1,19 @@
 import React, { useEffect } from "react";
-import { APIDictionary } from "../types";
-import { ProgressOverlay } from "../../../utils/components";
-import { useQueryParams } from "../../../utils";
 import { useHistory, useLocation } from "react-router";
-import qs from "qs";
-import ViewDictionaries from "../components/ViewDictionaries";
-import {
-  retrievePublicDictionariesAction,
-  toggleShowVerifiedAction
-} from "../redux";
+import { Link } from "react-router-dom";
 import { Fab, Tooltip } from "@material-ui/core";
 import { Add as AddIcon } from "@material-ui/icons";
-import { Link } from "react-router-dom";
-import { ContainerOwnerTabs } from "../../containers/components";
+
+import ViewDictionaries from "../components/ViewDictionaries";
 import { Header } from "../../../components";
+import { ContainerOwnerTabs } from "../../containers/components";
+import { retrievePublicDictionariesAction, toggleShowVerifiedAction } from "../redux";
+
 import { TAB_LIST, PER_PAGE, TITLE } from "../constants";
+import { useQueryParams, generateURLWithQueryParams } from "../../../utils";
+import { ProgressOverlay } from "../../../utils/components";
+
+import { APIDictionary } from "../types";
```

Comment:
It's not really a good idea to do this kind of refactoring in a ticket that isn't about refactoring, especially as it potentially breaks other PRs and tends to obscure what this actually does

Reply:
@ibacher Understood. Shall we create a separate code refactor epic in that case? We can improve the developer experience by introducing some order to the imports. Some suggestions:

* Separate imports of third-party libraries and our files with a single empty line (maybe when there are more than 3 total imports?)
* Sort third-party imports by order of importance
(eg: core imports like `react`, `react-router-dom` first, `material-ui` and `redux` next, and util functions from `lodash` next)
* Sort named imports within curly braces alphabetically.
eg: `import { applyTransformation, calculateTotals, getFormulation } from "some-source";`

Reply:
I agree with you @merovingienne, I use that in some other project I work on and it is indeed a good coding convention.

Reply:
@merovingienne 100% approve of adding an epic to do that. I might suggest using something like [this prettier plugin](https://github.com/trivago/prettier-plugin-sort-imports) so we can automate this across files.